### PR TITLE
Migrate regression and other tests to the new pytest marker

### PR DESCRIPTION
**Pull request by ljvmiranda921**
We now have a new @pytest.mark.issue decorator that we can use to label tests related to a specific issue on the issue tracker.

## Description

We now have a new @pytest.mark.issue decorator that we can use to label tests related to a specific issue on the issue tracker. I added this marker to all regression tests and others that reference a specific issue

### Types of change

Added pytest markers

## Checklist

- [x] I confirm that I have the right to submit this contribution under the project's MIT license.
- [x] I ran the tests, and all new and existing tests passed.
- [x] My changes don't require a change to the documentation, or if they do, I've added all required information.

### CODE REVIEW

**Comment by ljvmiranda921** [comment]
Lol it's a lot :O will probably automate the boring stuff haha

File: spacy/tests/regression/util_add_marker.py [hunk 1]
```diff
@@ -0,0 +1,32 @@
+import re
```
Comment (ljvmiranda921) [inline spacy/tests/regression/util_add_marker.py:1]: I am not sure if this should even be committed... 🤔

**Review by ljvmiranda921 — COMMENTED** [review]
A few comments

File: spacy/tests/regression/util_add_marker.py (hunk 1 above)
Comment (adrianeboyd) [reply spacy/tests/regression/util_add_marker.py:1]: No, this is only going to be run once, so I don't think it should be committed.

Comment (ljvmiranda921) [reply spacy/tests/regression/util_add_marker.py]: Removed in https://github.com/explosion/spaCy/pull/9629/commits/909177589dcdbde1cd4770f9f744d4d57d08d7e0

**Review by svlandeg — APPROVED** [review]
Thanks for taking this on Lj, looks good!

A related task we want to do, is move most of the tests out of `/regression/` and add them to the modules where they belong, but we could do this as a separate PR?

**Comment by ljvmiranda921** [comment]
Hi @svlandeg , awesome thanks! Yeah sure, been meaning to do that in another PR. Might take some time though to sift through each regression. Will do that as well! 💯

**Pull request merged**